custom-connectors/EleadCustomerSales/apiProperties.json

```diff
@@ -0,0 +1,8 @@
+{
+  "properties": {
+    "connectionParameters": {},
+    "iconBrandColor": "#1F1A3C",
+    "capabilities": [],
+    "publisher": "Bhargava, Aman Avinash"
```

Comment:
The `stackOwner` property is missing. Additionally, for a certified connector, these fields should not contain PII, but instead point to the wider company the publisher and stackowner is.